app/templates/googleTagManager.html

```diff
@@ -0,0 +1,7 @@
+<div class="text" style="height: 115px" ng-controller="gtmPluginCtrl">
+    <span class="pluginLabel" ng-class="{disabled: plugin!=null && !plugin.enabled}" id="{{property.id}}">{{property.label}}</span>
```

Comment:
can plugin be undefined ?

Reply:
this logic was copied from other templates I didn’t get into it